static/scripts/orbit-camera.js

```diff
@@ -240,7 +240,7 @@ OrbitCamera.prototype.initialize = function () {
 
 OrbitCamera.prototype.update = function (dt) {
     // Add inertia, if any
-    var t = this.inertiaFactor === 0 ? 1 : Math.min(dt / this.inertiaFactor, 1);
+    var t = 1.0 - Math.max(0.0, Math.min(1.0, Math.pow(this.inertiaFactor, dt)));
```

Comment:
```suggestion
    const t = 1 - Math.max(0, Math.min(1, this.inertiaFactor ** dt));
````

Reply:
How pedantic am I? 😆

Reply:
Yes **very** :) This camera script isn't run through the same build process as the app so const won't get converted to var.

Reply:
Oh ha, I didn't notice that. But then, shouldn't we avoid diverging `orbit-camera.js` from the original? If this change is an improvement, shouldn't it be made in the engine script first?
BTW, the viewer doesn't currently work in IE11....and I suspect it will be extremely hard to make it work there.

Reply:
reverting this change. will make the change to the engine script in future, add scripts to the engine npm package and get rid of this script altogether.